odk1-src/collect-offline-maps.rst

```diff
@@ -1,44 +1,68 @@
 
 .. _offline-maps-quick-start:
 
 Offline maps quick start
 -------------------------
-#. From :ref:`Collect's settings <mapping-settings>`, change the Mapping SDK to Google Maps SDK
 #. :ref:`Get or create your MBTiles file <getting-map-tiles>` with `TileMill <https://tilemill-project.github.io/tilemill/>`_ or other software.
-#. :ref:`Transfer tiles to devices <transferring-offline-tiles>`. The MBTiles file must be placed in a sub-folder on your device in the :file:`/sdcard/odk/layers` folder.
-#. Open a :ref:`question that displays a map <location-widgets>`.
-#. Tap the layers icon and select your map
+#. :ref:`Transfer tiles to devices <transferring-offline-tiles>`. The MBTiles file must be placed on your device in the :file:`/sdcard/odk/layers` folder.
+#. Select your offline layer in the :ref:`reference layer settings <reference-layer-settings>`.
+#. Open a :ref:`geopoint <geopoint-maps>`, :ref:`geotrace <geotrace-widget>`, or :ref:`geoshape <geoshape-widget>` questions.
+
+Your offline layer should be displayed. If it has transparency (PNG or PBF tiles only), the selected basemap will show through. If it does not have transparency or you are offline, only your offline layer will be displayed.
 
 .. _getting-map-tiles:
 
-Getting map tiles
+Getting map tilesets
 -------------------------
-`Open Map Tiles <https://openmaptiles.org/>`_ hosts many free map tile files that can be used in ODK Collect.
+`OpenMapTiles <https://openmaptiles.org/>`_ hosts many free map tile files that can be used in Collect.
 
-To create MBTiles files, use one of the `compatible applications <https://github.com/mapbox/mbtiles-spec/wiki/Implementations#applications>`_ that can write tiles. Commonly used free software packages are `TileMill <https://tilemill-project.github.io/tilemill/>`_ and `QGIS <https://qgis.org/en/site/>`_ with the `QTiles plugin <https://github.com/nextgis/QTiles#qtiles>`_.
+To create MBTiles files, use one of the `compatible applications <https://github.com/mapbox/mbtiles-spec/wiki/Implementations#applications>`_ . Commonly used free software packages are `TileMill <https://tilemill-project.github.io/tilemill/>`_ and `QGIS <https://qgis.org/en/site/>`_ with the `QTiles plugin <https://github.com/nextgis/QTiles#qtiles>`_.
 
+If you have existing geospatial data that is not in an MBTiles file, you may be able to convert it for use in Collect. For example, `Tippecanoe <https://github.com/mapbox/tippecanoe>`_ is a tool to build vector MBTiles files from GeoJSON features (see warning above: vector MBTiles files are only supported with Mapbox basemaps and are displayed without styling).
+
+The name of the tileset provided in the MBTiles file will be displayed in the Collect layer selection menu. The filename will also be displayed.
```

Comment:
A link to the spec could be good here.

    The choices in the Collect layer selection menu will show the
    name of the tileset (from the
    `Metadata <https://github.com/mapbox/mbtiles-spec/blob/master/1.3/spec.md#metadata>`
    table in the MBTiles file), as well as the path to the file.

Reply:
Oh, as I read further, I'm thinking this paragraph should be moved down below at the end of the "Selecting offline tilesets" section.